tests/benchmark/test_worst_bytecode.py

```diff
@@ -102,7 +104,9 @@ def test_worst_bytecode_single_opcode(
 
     # Set the block gas limit to a relative high value to ensure the code deposit tx
     # fits in the block (there is enough gas available in the block to execute this)
-    env = Environment(gas_limit=code_deposit_gas_minimum * 2 * num_contracts)
+    minimum_gas_limit = code_deposit_gas_minimum * 2 * num_contracts
+    if env.gas_limit < minimum_gas_limit:
+        env = Environment(gas_limit=minimum_gas_limit)
```

Comment:
I think we could simplify to this:
```suggestion
    env = Environment()
    assert env.gas_limit >= code_deposit_gas_minimum * 2 * num_contracts
```
If this assertion is violated, probably means it needs some attention. Being discovering that for whatever reason this `code_deposit_gas_minimum * 2 * num_contracts` is wrong, or that we need to bump the `BENCHMARK_MAX_GAS` constant. Since at EOD I think the hole point of having this huge `BENCHMARK_MAX_GAS` is to always cover all setup costs (if not, it seems a bit handwavy what "max" means, since here we would have an exception.

Note this is a nit -- I don't have strong opinions. Mainly a bit of tension about what "max" really means, and I prefer to lean towards someone double-checking why this happened.

Leaving it up to you!

Reply:
I've put a more comprehensive exception message to make it clear that the root cause of the issue is `BENCHMARKING_MAX_GAS`.

I agree that we have to keep updating `BENCHMARKING_MAX_GAS` so it's enough to cover the requirements of all tests' setups.

Thanks for pointing this out.